pkg/ssh/ssh.go

```diff
@@ -333,8 +333,8 @@ func (l *SSHTunnelList) delayedHealthCheck(e sshTunnelEntry, delay time.Duration
 		defer runtime.HandleCrash()
 		time.Sleep(delay)
 		if err := l.healthCheck(e); err != nil {
-			klog.Errorf("Healthcheck failed for tunnel to %q: %v", e.Address, err)
-			klog.Infof("Attempting once to re-establish tunnel to %q", e.Address)
+			klog.ErrorS(err, "Healthcheck failed for tunnel to ", e.Address)
```

Comment:
Error formate,  the parameter ` e.Address` should be format of key-value. It's better to start before read  kubernetes/enhancements#1602.